extension/gulpfile.js

```diff
@@ -88,8 +88,11 @@ gulp.task('browserify', () => {
   ], {read: false})
     .pipe(tap(file => {
       file.contents = browserify(file.path, {
+        fullPaths: true,
         transform: ['brfs']
       })
+      // explicitly require a few things traceviewer will need
+      .require(['gl-matrix', 'jszip/dist/jszip.min.js'])
```

Comment:
is this needed? why aren't these picked up from the `require`s in `input-readiness-metric.js`?